.eslintrc.js

```diff
@@ -3,6 +3,10 @@ module.exports = {
   extends: ['@storybook/eslint-config-storybook', 'plugin:storybook/recommended'],
   rules: {
     '@typescript-eslint/ban-ts-comment': 'warn',
+    'jest/no-standalone-expect': [
+      'error',
+      { additionalTestBlockFunctions: ['it.skipWindows', 'it.onWindows'] },
```

Comment:
This gives us a way to write tests that don't run on windows or only run on windows.  There were only a handful of times that I needed this, and maybe there are other ways around it, but it seemed like a useful enough utility that I thought it would be worth adding.